### ISSUE TITLE
request to make the Guiding Assistant window resizable

### ISSUE THREAD

**Issue by agalasso** [OpenPHDGuiding/phd2#513]
and also make the table columns resizable

![image](https://cloud.githubusercontent.com/assets/6864470/13404394/6c131f40-dee7-11e5-995d-cf7feeaebf86.png)

**Comment by bradleyjdavid** [comment]
I agree. The text `"Guiding output is disabled and star movement is being measured.  Click Stop when the RMS and polar alignment values have stabilized (at least 2 minutes)."` gets cut off.
